### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mimansha-swarup**
Initial Pull Request for Landing Page

### CODE REVIEW

File: landing.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    color: var(--Green-0);    
+}
+
+.hero-landing{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 10rem;
+    padding: 1rem 3rem;
+    height: 100%;
```
Comment (vijaytembugade) [inline landing.css]: it would be great if you give this like height: 100vh.

**Comment by vijaytembugade** [comment]
I think for landing page, It would be great if you add preview like screen shot or something link for this. It will ease the
 reviewing process. Seems  like all code is fine.

File: landing.css (hunk 1 above)
Comment (mimansha-swarup) [reply landing.css]: will do that

Comment (mimansha-swarup) [reply landing.css]: when I am giving height: 100vh. 
 it gets overflow so I made it as 80vh is that fine?

**Review by srishti-maurya — APPROVED** [review]
Simple and elegant landing page

**Review by vijaytembugade — APPROVED** [review]

**Pull request closed without merging**